src/SpectatorView.Native/SpectatorView.Compositor/Compositor/VideoEncoder.cpp

```diff
@@ -16,7 +16,7 @@ VideoEncoder::VideoEncoder(UINT frameWidth, UINT frameHeight, UINT frameStride,
     audioChannels(audioChannels),
     audioBPS(audioBPS),
     fps(fps),
-    bitRate(62 * 1000 * 1000 + 500 * 1000), // 62,5 MBit/s
+    bitRate(300 * 1000 * 1000), // 300 MBit/s
```

Comment:
The bit rate should change when you swap to 1080p?

Reply:
In my latest update to the PR I've switched this back to 62.5/level 4.2 when using 1080p encoding, although testing doesn't seem to indicate that this produced higher-bitrate files.